client/my-sites/checkout/checkout/domain-details-form.jsx

```diff
@@ -344,117 +156,46 @@ export class DomainDetailsForm extends PureComponent {
 	renderPrivacySection() {
 		return (
 			<PrivacyProtection
-				checkPrivacyRadio={ this.allDomainItemsHavePrivacy() }
```

Comment:
It seems the `PrivacyProtection` component was not updated after this rename and it's still relying on `this.props.checkPrivacyRadio`, which leads to this interesting state:
<img width="745" alt="screen shot 2017-12-03 at 11 09 36" src="https://user-images.githubusercontent.com/3392497/33524295-771c60f8-d81a-11e7-9a55-83e8517bf95e.png">

Reply:
How can I reproduce this? 

FYI, I ditched most of the props on this component since `countriesList`, `disabled`, `fields`, and `onCheckboxChange` weren't being used by `<PrivacyProtection />` at all. 

And `this.handleCheckboxChange` didn't even exist in the domain details component :)

Reply:
IIRC, I did not do anything special - just try to register a new domain that supports privacy. And try to check a different radio button - both will be checked as a result.